codalab/rest/bundles.py

```diff
@@ -678,6 +680,7 @@ def _fetch_bundle_contents_blob(uuid, path=''):
     else:
         response.set_header('Content-Disposition', 'attachment; filename="%s"' % filename)
     response.set_header('Target-Type', target_info['type'])
+    response.set_header('X-Codalab-Target-Size', target_info['size'])
```

Comment:
Should we put a comment that this is the uncompressed size, whereas the file will be compressed?

Reply:
Oh, this is Target-Size, so I guess it's fine.  But how does the client know how to interpret this header?

Reply:
I added a note to the docstring in this method describing this fact of X-CodaLab-Target-Size.